File: static/data/mpi1.c
#include <stdio.h>
#include "mpi.h"
int main(int argc, char *argv[]) {
	int rank, npes;
	MPI_Init(&argc, &argv);
	MPI_Comm_rank(MPI_COMM_WORLD, &rank);
	MPI_Comm_size(MPI_COMM_WORLD, &npes);
	if ( rank == 0 ) {
		printf("Running on %d Processes\n",npes);
	}
 fflush(stdout);
	printf("Greetings from process %d\n",rank);
	MPI_Finalize();
}

File: static/data/mpi1.f90
program hello 
use mpi 
implicit none
integer ::  myrank, nprocs 
integer ::  err 
call MPI_INIT(err) 
call MPI_COMM_RANK(MPI_COMM_WORLD, myrank, err) 
call MPI_COMM_SIZE(MPI_COMM_WORLD, nprocs, err) 
if ( myrank .eq. 0 ) then 
   print *, 'Running on ',nprocs,' Processes' 
endif 
print *, 'Greetings from process ', myrank
flush(6)
call MPI_Finalize(err)
end program